mytoyota/controller.py

```diff
@@ -22,7 +22,7 @@
 
 _LOGGER: logging.Logger = logging.getLogger(__package__)
 
-CACHE_FILENAME: Optional[str] = expanduser("~/.cache/toyota_credentials_cache_contains_secrets")
+CACHE_FILENAME: Path = Path.home() / ".cache" / "toyota_credentials_cache_contains_secrets"
```

Comment:
Why do we store the credentials?

Reply:
See discussion here https://github.com/DurgNomis-drol/mytoyota/pull/242#issuecomment-1843433435 and here https://github.com/DurgNomis-drol/mytoyota/pull/242#issuecomment-1843750651